tests/jenkins/nrnivmodl-core.sh

```diff
@@ -0,0 +1,8 @@
+#!/usr/bin/bash
+
+. /gpfs/bbp.cscs.ch/apps/hpc/jenkins/config/modules.sh
+module load intel/18.0.3 hpe-mpi/2.16
```

Comment:
remove versions `/18.0.3` & `/2.16` so that if new modules are available, we dont have to change this.

Reply:
Should the version definition be removed from `pgi`, `cuda`, `cmake` and `boost` as well?

Reply:
yeah...I would remove them now. I believe we would be deploying compatible ones.